packages/puppeteer-core/src/cdp/Target.ts

```diff
@@ -102,23 +102,30 @@ export class CdpTarget extends Target {
 
   protected _sessionFactory(): (
     isAutoAttachEmulated: boolean,
-  ) => Promise<CDPSession> {
+  ) => Promise<CdpCDPSession> {
     if (!this.#sessionFactory) {
       throw new Error('sessionFactory is not initialized');
     }
     return this.#sessionFactory;
   }
 
-  override createCDPSession(): Promise<CDPSession> {
+  /**
+   * @internal
+   */
+  _createCDPSession(): Promise<CdpCDPSession> {
```

Comment:
We don't need to wrap this into a internal function, as CdpCDPSession extends CDPSession, so TypeScript won't complain.

Reply:
Done